src/services/config.ts

```diff
@@ -53,8 +53,9 @@ class Config {
   }
 
   getLocalConfig() {
+    const basePath = process.env.BASE_URL || './'
```

Comment:
@vykulakov This assumes that a `BASE_URL` always has a trailing slash. This is the same assumption as for `index.html`.